tests/SessionTest.php

```diff
@@ -12,6 +12,16 @@ final class SessionTest extends TestCase
 {
     private ?Session $session = null;
 
+    protected function tearDown(): void
+    {
+        if ($this->session !== null) {
+            $this->session->destroy();
+        }
+    }
+
+    /**
+     * @runInSeparateProcess
```

Comment:
You can use `@runTestsInSeparateProcesses` in the class annotation instead of all `@runInSeparateProcess`

Reply:
done.